webcompat/static/js/lib/bugform.js

```diff
@@ -292,11 +297,13 @@ function BugForm() {
 
   /* Check if Browser and OS are empty or not, only
      so we can set them to valid (there is no invalid state) */
-  this.checkOptionalNonEmpty = function() {
+  this.checkOptionalNonEmpty = function(types) {
     _.forEach(
-      [this.browserField, this.osField],
-      _.bind(function(input) {
+      [...types],
+      _.bind(function(key) {
+        const input = this[key];
```

Comment:
Can we change this to `var` please? We're doing ES5 until we decide we're ready to ship ES6 (or transpile -- but that's probably a future discussion).